packages/forms/resources/views/components/wizard.blade.php

```diff
@@ -34,13 +34,15 @@
             this.step = this.getSteps()[previousStepIndex]
 
             this.autofocusFields()
-            this.scrollToTop()
+            this.scroll()
         },
 
-        scrollToTop: function () {
-            this.$nextTick(() =>
-                this.$root.scrollIntoView({ behavior: 'smooth', block: 'start' }),
-            )
+        scroll: function () {
+            this.$nextTick(() => {
+                this.$root.scrollIntoView({ behavior: 'smooth', block: 'start' })
+
+                this.$refs.headerSteps.children[this.getStepIndex(this.step)].scrollIntoView({ behavior: 'smooth', block: 'start' })
```

Comment:
Following the header CSS class, this can probably just be header instead.

```suggestion
                this.$refs.header.children[this.getStepIndex(this.step)].scrollIntoView({ behavior: 'smooth', block: 'start' })
```

Reply:
Too easy, have pushed the change!